bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -31,7 +31,7 @@ def self.detected_bundler_version(lockfile)
         if (matches = lockfile.content.match(BUNDLER_MAJOR_VERSION_REGEX))
           matches[:version]
         else
-          FAILOVER
+          "1"
```

Comment:
❓ Should we remove the `FAILOVER` constant or change it just to equal `1`? Should it always be `1.173` instead? Are there any scenarios where passing the full major version would cause problems?

Reply:
We use it in two separate places, this `detected_bundler_version` method is used to gather metrics on the bundler major version, but in `bundler_version` we now need the full version.